### PR TITLE
Freebsd rts dtr

### LINKED ISSUE #81: FreeBSB: Manual controlling rts and dtr won't work
As I tested with my FreeBSD 11 installation, the current implementation in `term.c` via `tcsetattr()` won't properly. E.g. trying `C+a+p` results in:

    *** pulse DTR ***
    *** FAILED

`C+a+t` always prints:

    *** DTR: up ***

and `C+a+g` always:

    *** RTS: down ***

I'd tried out the alternate `ioctl()` which is currently in use for Linux and macOS and this also works for FreeBSD. So the fix seems as simple as just changing some `#if` by adding `defined(__FreeBSD__)`.

I'll also test the `term_get_mctl()` implementation about this.

### PR DESCRIPTION
**Pull request by JoeMerten**
Fixed #81

### CODE REVIEW

File: picocom.c [hunk 1]
```diff
@@ -1197,6 +1197,8 @@ do_command (unsigned char c)
         fd_printf(STO, "\r\n*** pulse DTR ***\r\n");
         if ( term_pulse_dtr(tty_fd) < 0 )
             fd_printf(STO, "*** FAILED\r\n");
+        else
```
Comment (npat-efault) [inline picocom.c]: Why?

Comment (npat-efault) [reply picocom.c]: never mind, I got it. It's ok.

**Comment by npat-efault** [comment]
Merged, thanks!

File: picocom.c (hunk 1 above)
Comment (JoeMerten) [reply picocom.c]: When entering this code with `dtr_up=0` it goes out of sync afterwards.
That's why I set `dtr_up=1` after successful calling `term_pulse_dtr()`.

Comment (JoeMerten) [reply picocom.c]: I'd considered to completely get rid of those both `rts_up` and `dtr_up` variables. But I think we still need it as long as `term_get_mctl()` has not been implemented for all target platforms.

Comment (npat-efault) [reply picocom.c]: Yes I agree. It's fine as it is.

**Pull request merged**